config/site_settings.yml

```diff
@@ -1100,6 +1100,10 @@ files:
     client: true
     regex: '^((https?:)?\/)?\/.+[^\/]'
     shadowed_by_global: true
+  restrict_letter_avatar_colors:
```

Comment:
Is there an expect format for the values here? Should it only be HEX codes?

Reply:
Yes, only 6-digit hex codes. I have added a validator for it.